yapapi/log.py

```diff
@@ -249,40 +257,45 @@ class SummaryLogger:
     ```
     """
 
-    logger = logging.getLogger("yapapi.summary")
+    # Define some messages a string constants here, so they can be used e.g. in
+    # integration tests, in assertions that expect certain messages to be logged
+    GOLEM_SHUTDOWN_SUCCESSFUL_MESSAGE = "Golem engine has shut down"
```

Comment:
Perhaps this should be pushed out into the `log` module itself (rather than be in `SummaryLogger`)? It's not strictly related to that specific class.

Reply:
I believe it is strictly related, this is the message that `SummaryLogger` logs when handling the `ShutdownFinished` event.